stl/inc/yvals_core.h

```diff
@@ -1226,9 +1226,14 @@
 #define __cpp_lib_constexpr_vector 201907L
 #endif // defined(__cpp_constexpr_dynamic_alloc) && !defined(__clang__)
 
-#define __cpp_lib_destroying_delete            201806L
-#define __cpp_lib_endian                       201907L
-#define __cpp_lib_erase_if                     202002L
+#define __cpp_lib_destroying_delete 201806L
+#define __cpp_lib_endian            201907L
+#define __cpp_lib_erase_if          202002L
+

```

Comment:
We should additionally mention the primary paper `P0645R10 <format> Text Formatting` in the CXX20 section of the comments above.